servers/src/test/java/tachyon/worker/block/TieredBlockStoreTestUtils.java

```diff
@@ -32,37 +38,45 @@
 import tachyon.worker.block.meta.TempBlockMeta;
 
 /**
- * This class provides utility methods for testing tiered block store.
+ * This class provides utility methods for setting and testing tiered block store.
  */
 public class TieredBlockStoreTestUtils {
   /**
    * Default configurations of a TieredBlockStore for use in {@link #defaultMetadataManager}. They
    * represent a block store with a MEM tier and a SSD tier, there are two directories with capacity
-   * 100 bytes and 200 bytes separately in the MEM tier and three directories with capacity 1000,
-   * 2000, 3000 bytes separately in the SSD tier.
+   * 2000 bytes and 3000 bytes separately in the MEM tier and three directories with capacity 10000,
+   * 20000, 30000 bytes separately in the SSD tier.
    */
   public static final int[] TIER_LEVEL = {0, 1};
   public static final StorageLevelAlias[] TIER_ALIAS = {StorageLevelAlias.MEM,
       StorageLevelAlias.SSD};
   public static final String[][] TIER_PATH =
       { {"/mem/0", "/mem/1"}, {"/ssd/0", "/ssd/1", "/ssd/2"}};
-  public static final long[][] TIER_CAPACITY = { {2000, 3000}, {10000, 20000, 30000}};
+  public static final long[][] TIER_CAPACITY_BYTES = { {2000, 3000}, {10000, 20000, 30000}};
+  public static final String WORKER_DATA_FOLDER = "/tachyonworker/";
+
+  public static TachyonConf sTachyonConf = WorkerContext.getConf();
 
   /**
-   * Create a {@link TachyonConf} for a {@link TieredBlockStore} configured by the parameters. For
-   * simplicity, you can use {@link #defaultTachyonConf(String)} which calls this method with
-   * default values.
+   * Setup a {@link TachyonConf} for a {@link TieredBlockStore} with several tiers configured
+   * by the parameters. For simplicity, you can use {@link #setTachyonConfDefault(String)} which
+   * calls this method with default values.
    *
+   * @param baseDir the directory path as prefix for all the paths of directories in the tiered
+   *        storage. When specified, the directory needs to exist before calling this method.
    * @param tierLevel like {@link #TIER_LEVEL}, length must be &gt; 0.
    * @param tierAlias like {@link #TIER_ALIAS}, each corresponds to an element in tierLevel
    * @param tierPath like {@link #TIER_PATH}, each list represents directories of the tier with the
    *        same list index in tierAlias
    * @param tierCapacity like {@link #TIER_CAPACITY}, should be in the same dimension with tierPath,
    *        each element is the capacity of the corresponding dir in tierPath
+   * @param workerDataFolder when specified it sets up the tachyon.worker.data.folder property.
    * @return the created TachyonConf
+   * @throws Exception when error happens during creating temporary folder.
    */
-  public static TachyonConf newTachyonConf(int[] tierLevel, StorageLevelAlias[] tierAlias,
-      String[][] tierPath, long[][] tierCapacity) {
+  public static TachyonConf setTachyonConf(String baseDir, int[] tierLevel, 
```

Comment:
not sure you need to return `TachyonConf`, because if some one needs to get it, he/she can always access it from `WorkerContext`. I checked the code, the only occurrence to use a returned value from `setTachyonConf` is in fact redundant.  So you may want to check if we can simply return void for this set of `setTachyonConf` methods.